lib/project_types/script/layers/infrastructure/assemblyscript_project_creator.rb

```diff
@@ -68,14 +34,6 @@ def write_npmrc
           )
         end
 
-        def write_tsconfig_file(dir, path_to_source)
-          AssemblyScriptTsConfig
```

Comment:
I think we can scrap the `AssemblyScriptTsConfig` class now too. I don't think its used anywhere else